scvi/inference/inference.py

```diff
@@ -78,13 +74,18 @@ def __init__(
         self,
         model,
         gene_dataset: GeneExpressionDataset,
-        train_size: Union[int, float] = 0.8,
+        train_size: Union[int, float] = 0.9,
```

Comment:
I don't have an issue with this as the default will be "good" but @romain-lopez what do you think about the bump from 0.8 to 0.9?